### ISSUE TITLE
migrate to new clustered db [blocked by ops work]

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#1359]
Julian wants prescat db to be migrated to clustered db instance;  when this is done we will at a minimum need to update our shared_configs settings for pres_cat, and coordinate with ops for prescat down time and deploying shared_configs updates.

**Comment by jmartin-sul** [comment]
leaving this open for whatever config changes we'll have to make.

@justinlittman can correct me if i'm getting this wrong, but i believe he did some investigation into whether there's easily available support for configuring hot failover for PG databases in a rails environment.  i don't believe justin found anything viable.  also, iirc, @julianmorley said that there are expensive proprietary solutions, but we don't want to use those.  in the event that prescat needs to failover to a different DB connection, we can live with a bit of downtime and some manual re-configuration.

**Comment by justinlittman** [comment]
No correction necessary.

**Comment by ndushay** [comment]
assigning to @tallenaz for now, but may also need a dev when the time comes.

**Comment by tallenaz** [comment]
If we're not changing storage roots at the same time, no update is necessary to `shared_configs`, instead:

- [ ] shut down any operations that would create, update, or delete records in the db.
- [ ] run puppet to install postgresql clients and update `database.yml`
- [ ] dump/restore database from old environment to new environment
- [ ] re-enable operations

**Comment by tallenaz** [comment]
Prescat stage is now pointing to the new db setup. I'll start building out production, and keep an eye out on stage for any issues.

**Comment by ndushay** [comment]
As Tony reports, this is already done for stage and he expects to do this in Prod the week of 2/18/2020

**Comment by tallenaz** [comment]
This is done in prod.
